Fix:

error: implicit declaration of function 'atoi' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'inet_addr' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'inet_ntoa' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'random' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'sigaction' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'sigaddset' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'sigemptyset' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'srandom' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
error: conflicting types for 'malloc'
--- bing.c.orig	2001-01-19 12:28:53.000000000 -0600
+++ bing.c	2025-08-22 02:16:02.000000000 -0500
@@ -174,6 +174,9 @@
 #include <errno.h>
 #include <string.h>
 #include <limits.h>
+#include <arpa/inet.h>
+#include <signal.h>
+#include <stdlib.h>
 
 #include "sysdep.h"
 
@@ -1233,7 +1236,7 @@
 	int i;
 	int ch, hold, recv_packlen, preload;
 	u_char *datap, *recv_packet;
-	char *target1, *target2, *malloc();
+	char *target1, *target2;
 	u_char ttl, loop;
 #ifdef IP_OPTIONS
 	char rspace[3 + 4 * NROUTES + 1];	/* record route space */
